moped-editor/src/App.js

```diff
@@ -48,6 +49,8 @@ const App = () => {
     cache: new InMemoryCache(),
   });
 
+  console.log(`🛵 ${pckg.name} ${pckg.version} 🛵`);
```

Comment:
git rid! 🙏

Reply:
Sorry, I disagree. I want to keep this easily available until we decide where it should live in the app UI.